{"base_commit":"cb8286d26f8048e5eced04cd4e70cedca5525398","message":"feat(creative): ship immutable Prompt Library with scoped retrieval and paste-save UI","encoding":"zlib-base64","hashes":{"docs/2026-09-21-higgsfield-creative-engine-progress.md":"10c5963aa1b2c43432f29b1ed70a5b9e233e0988","engine/creative_models.py":"8459162000feb7a099e9a4926aede1ae92d943b9","engine/prompt_library.py":"25318159e66aedec6f1cdad011e3e5a6117753fe","engine/prompt_views.py":"70a523b42d55e006946ca6e6b0121bf109cdbcd1","engine/static/css/creative-screen.css":"6d1959172157ea5dde7290f7d1170309355a4a63","engine/static/js/prompt-library.js":"96516f3468e9c76177234cd15f65692a09a74049","engine/test_creative_library.py":"30051a34d5fcac138a4f3c27e80e11345054be20","engine/test_prompt_ui.py":"f1edbd6b81eb3448b23f832543713a3242bf717b","engine/urls.py":"e8907f70a357f0403e882f7076d7003d6b92dee5","templates/base.html":"0ba4d38ad339984d32ac5a2ca3a35d1758d98ae8","templates/components/navigation_links.html":"69567222b155d774eac9897ab8d49148a1074096","templates/engine/library_tabs.html":"d47b5a676c6a489a27b747882bce5a9a2b5dac5f","templates/engine/media_job.html":"85d140a9420207415f0f9efbf19633ae98a32945","templates/engine/media_library.html":"7abb6098d0abbd6b50e69bf23d104d92f8079f0b","templates/engine/prompt_bulk.html":"d26f3ec2a9aa726d1944593232b7ca9b975272a0","templates/engine/prompt_detail.html":"a93aed34cee72a93299c3bedb0b01099f26716d6","templates/engine/prompt_extra_fields.html":"bd004d7a978a35f88aba96fa47bd9265946327ba","templates/engine/prompt_library.html":"a12549bdeaf9d248fe643d06f1965644888acf7e"}}
--- PATCH ---
eNrNvdliG0eWKPiur4iCpxuACYDYF0pQWZbkstqW5RJl1+0r80KRyACYIpAJZyZI0So+zv2AeZi5H1D9MD/Q8z7+k/6SOUtEZuQCEKTk7qluU8jMWE+cPU6c
cL3FQjSbSy8W8tgN5tFxt90dNtuTZrfTPPeWy2jhqZXbnIdKxt6laip/6fmquQmDZaiiqLV2hXPPig8831UfRGcyb7f7Hac76fUnai6HatjpDofdcV/KiZr0
OoN+rzuSqtXqtOeDybAnJRSe93vwetGdOB3ljtpy4ExUt9dT7cl4LDrt9rDff9BsNu89qwdHR0f3n9lXX4lmp9HpiSP42xdfffVAfCGe/vT69fMf3ojTN0/e
PH8gYHjiexmrKBbzYL1ZqVi5IorV5kT8GMKLWHzvOaEMr4V0XQ97EtH8XK1lQ6y9ZQh9B76QvitCRf3iI7YWiUsVejBGV9Q63Ta/q7ceHB3aXaTCS2+uqO2f
Xtit9ce6tYciWCxWMGvx9Bwqe9u1WMnrYAttn6v5BRSWsej0++3NB2qmN4FfrQeiKb4OpT8/PxHvFgC9baiO5+cyXm7i5jzwY+XHBprr+eadPebo3NtsoF2A
srud0xzWXgztdAbz0VCqxUS2x30FWNCZLyYjNZ+743G7M+rKwbgzGLe770SNwfdQIFRO//q9F/Mc8fHHIIoBivBWPH2hwbiRUaTcOg9bRormm/brQK/tnuz1
u3IymLdVvzectIfdgQMzA/ycq1FXtfvtd1T/pfR8WKiFCpU/V1AdRjfo9vvDkVp0O8P5YtDtuF01XsDgHXfQH/U7yum58xFXf6Y2q+D6RPzw6g1AWPpLgEQQ
pksDjcfnXgRLR3jAQ14FsBLhiXitgM5CGLnvq3kM1c5lJPwACq/gGWpfBeFFtJFz1RIvYrHycPLvXl6Lv5n3ALt3sZJNtz0ESpUrb3I5H/WcXxf+uP2u3hAO
LLwHlXANQwmrE6pftx4gpVAfNitvDtxlG/EQFl64ZtR11CIIFX5oiScR4sy/BL6MHlIrnr9VAvnDBgfvY4ubIPJg9Nc0XJrh98FcroTyL70w8NdQ6iRBbYHj
wDYjX26i8yAGhAWgz1Wn4y5GUnadQWfR7fYGsj3qjDuTUXs4V7LrDkewdN13Deh7vtq6nr8U0dZZA86tgPyCbThXMEIvClYSIffjdXwOU+m1Or3WQMA44q1c
wYDElRefJzRyda7UKmqJHwKDvjh/4B84M0+uomNXxrKVIhmgICFmZyJWNMcUIVvINn5QH2IALUI6ltHFifBw1giBYyKWDRN0JC9hAjCX9XobSwdmEIQeUBc1
+CFuEIA/KOQgceipS7lqwAdfAridYOu7El5GNJIch/jpBZHm7mEANgK44hCmCjTuiqeaVwL5e2rRQMRFPKWul4Bv4XWDls5bAZZgh0mFZ4BGhLQEAlr2H6Xn
iqXyFfPA6ET89+evXz1McU1uAAAwG1gng4muQTfpX8M7+LbBVqJ1cKGoaWgZePQX4muACa66gwMlPt7tNkbiqNuDv/D45lwxLkfbDXQHDeNUF9vVKp29xm9s
ASnz3Qv/AokqlC7BvTaqt+IP8TsBhbo4obfiTPy0cQmjco1sVpKZvAzn58CwCJ4i3AKaLCNT9zQFtBFKunesSYz1Q7Kk16bWz8kaAHorGD4UW628JTKohgiB
mauwoVHpeCNDuQa6CpNlwk8K8Ht5HiNKYos5LNlgByBhuD2DYlqsIP68/XDHOnrkrxYLWGd4ffrsOxCOchNrrFlIb9Wcr4IIIREgIQSAEcmMn21DogJ3i4gC
4G5GcqGA4S3U/Hq+gh6vlHMeBBcNrIggPgbkCwBZr6n5113DXwBuCfDPJQL+JybZl09/JEBq+dwQwRXgKQovRmDWDPq9BqoG/X5jOECkOo/jTXRyfAxq2PnW
aUEnx5aiIT37aQ5Ih3QeKnUMYs/tgMY0Gk26k9Gw60wWw4kcScLkNyBpIxA4WkrMGaEYdRDZnxDWNhewmJZsiE0tOZ97yJ/kanUtQMTHiJ3vXl35r7YxDFC1
EDShq9yZjGtyGwczP7iagaoyfRNuVf1dCwWQ0wRZc54dwVxuAWdAXtgk6ZBaIK5AMkn3En4r9yFRlgsyc04YvfVlHEsQ4a4WwlTaDxLdoCXeBEJeBh5yFSBR
wBDvN6TlrW9k5kot5fxa4Jwj5HXrTUBo9y5R5gCpAddV63q9egeNXyEzWSEbDPwVYoHmd2vpA9ZEoLZofWcj4/k5oOzWj1L1LDpGfIhiCXJhc0xMwtI7LJ2D
cKPB5IojQbYr/uIZNUcEznuAAkkVVLK2yCxwZinoWsSZYB3OUabMiZdAA99uHWtx5Qro3QA4IviqD0Bu2JTWx8xKyEWcQ528/IoUDDVm2QXsidiiy3PAhlPg
orxerACUVx4AwAH2pdZAVKQjwrq4ICZbWpazhMUCoC9Fup0I54IzA+kVa+52Da9BBj04QkQH2SO6zl49NhF8MAHkokjAVPsL4J4w8ejB0bNXPzwXgJCm3kND
06mo0cqpTeF6rFAJWSQqHjAaM7QvxDeIaUZle3D0jhs4Zq46W/FIW5tr0DjMN4OKM2LLUfYjQmOWlMjUvw2bG6wirpS7VGEywL+B+p2olMegqlw/OHrOIh2F
xoYo6VIzs9NvnzS7gyFy6SZyQkS/hJlaoG0k8D42xbQO4q28+BpUNoDzHHVYXwsfUFV4of66BW57qjSOq0ZGCVRgBxEHnwcb0BgWQOyAEkTKcon0g0LyUgHX
jgDdoXK6dDi6S9RtACFIOUu0BZIBqXwDfhdcobgGrdR1SbdmJWm9XcUeMk9Q8ZCm5gpwDYcM+J1RoeaSLBWA64KMNsBbYHd+tPH0UICFoBRh1Sq+CgSSkAPK
n/gVZu/hfLJD/Pb5T69fnL558fR4Cwr2lghiJR1UFpdElmrtKBf1lgjNAlJtJED/GmQpkeZ8Jb01csivt6sLGMsGmEgEjcear5ElmfYHK+IB29FSNVHmJVIE
YMSlp65EHACxIKyA8ySYB4or8kxv6QdYgQidoIVcjOcLXIIJG6QHqnY2GzwHxSw+J5ZKSlaCpT/bAuRYi1LQIF4/f5aQebSFEZ+IzoDUAGRprDU7CiQOwJZU
Ul3WBQCh5CDuR7rsX14/f/4DQMZDhbzQ5lAbg1T0+QcQkgmGNx0AaZOxlSZIepZym7iW13oMEjGJKr/bsMkAAkQuFVAulXgHffRGuhPUGcetTn+ELDbwXUCH
6BrWfM3GNbCTlZL+jtbW8kKl8kc0WUDBv2543QTpBD2B6ccEH8FCx2QCUmOnZB4L5hlGWwD1tTcYdtv9yWQ87uE4jUvBjBaK5YxpeJMu6kMtMUkRMGITfsQo
yVvAs4lDucdG7BuRd7hh/xBnpI1clwxlMkGASQGXWdPkGItg4VSIQuAyi04XgLM+GAvRVgFWneq1J8sLaMMY2dRNuf2WEB2SCtkpSCmIYM7WW8XiWgE30SNM
jO2sNWxEHoh30BfIBmmJb+HHVYiqFwoYAB5I9oT1rIHNIJZ7rAQh/RouYPCF9Anp42eUOB5QgQwdL2ZVGxZ5cx4i26HVwB4NpHKW3YOjFxnLbhsRF87r7oCC
1CWg7RZGfX2M8DeGm275Ka0uQJmQ7S7+m5YwK4OODmD4gCsornFAPo53FSw9VCK0VNSIRD1besL8pGjN5sS8JSt2KgmrbGXDMCzNAHAgUk3Pbxpbtnkab10v
yGgPzEIjbS/TZwT4gfoDMuIyBUF/3nrZb9swr05EqE/Nj+dRlKgdzQh+Kb8F74oF30e666ZRPd5TKdQ90CMSHZcrOOcxKSDv9GtgO8Ci82+RkebfKeS1M7J8
ouSbbnUGQjN5meix74DiPVnomd++D5zkTTpmFL0lr1F1AX3ZBx3Xl5feklYMGvYvkl4P1ah+lBFZm5eo27rk0ohQgkqxADl5riWyUe9Z4OJkQCWR7Bt6kmiY
IA1AvkLbnXZz7flgpguEXCKYYUQkgqj9jD1Afgt2CbBSZgywGHjDPMOPWuKV0ZixQqJ4kVKIGoXrgf4gr9HDpse1ukYKRRWsIS48Eluko0QZRQ01N3JKGd2A
qAcA01zAijctIkxkNhvYEdRUCaG1xEtcUHJlGlKMtg5ihID65wr5ArBENN7nYXBFXDFj76wDBy3fdGUBxmCeoJbKohP7//HV6Rv2R52+/gYlmRYg2sFG0AB4
qwh0PlT5oKSvLrFzsuLgTYSaH/lnsNFDFJqTHTqCYNJqZSkcJXKHNJ4taly2skMunBJV5xsvhNZY4QGTWuFMQPUjrwmAbOsT+jSJeaswBGbH6qHnw3qh04zR
Lw4ulP8QdFxUjh2Qk/hRm6UaA7Hpa2SXa0Rgxh3QGUKatedrDXB1bdQBGjaKkSp7jmH+5PFFpHloFEbFmE3tUAWQUqiSc2cABNDToexqxbbILVpXf5zRunrD
jtG6Hv5RWhe1keybGCdxyOpBsGD3ZJZyn72HmoH49s3L79kbuGGjC+H49PTU7LZ86LTbbbPh8mHc71O3aDf/FqAnh8gBTXF03nlrIhrkSc42jqGlSy/yHHSD
oaKDwpxWH5SZUxIJ6EaPsNCW9CJmN6gXeWsSmLx8TIV6KwCtqK+B/JDx+CpG3Uf75ISD2xNAvEGwcSTAKiVEYHABc1Yv3UBAI0VDSu802Upcg1Ug4XBfx8gl
oEsAaTMOwGp32cdr4Mg+NK3GumA+fQAu4RmyKYKf5CzB1VhqCSG/Znw7TnmDPa4HR+zX2IJd0cg6Bok6SUM6RuZiDAX0Veh5MqQN+pHzP9FZcLqgVWp9l0mC
FUx206AyypY1KrEp3/kuo+qiyyjxtGvtBLmHZA3X+GvIetEeITTjmLEQQ6Htmg37eFAPvSrRc3mfyVZyt76WNZZyXlA5vwbVmVkqcb/4Gjmsw/sH2sZKBJn2
cTf0nNNNBXpuRhs1xzXZ7c2mfhKHNrkefZy3LYFc7T62ZJSRRnnt9kckjmAbJduW0T3U3Z/N7l7ikyWoLbfcOTQ2d8bd8dDtDhfjdn+sBgrsp3Z/7vbVqA0/
53Iw6A56kzE09nQLzNOPc8qx5ZJjZ65e5acvMmag3kbQXkTjIWSDDY2CZP7lPmc2jgHd8n4CMJ0CdDSDOQOg8n5TTbOwv8E4SS8b9nn6G3SiAHMkbZ8RFv2j
LsYvXJ2jJAexvtLUhxqGUOS3Z4xyYXrYLDm70RJFdyn6RtBbHCEjR0pLvcY8buhuFaA1xV5J8hoTC9X+Cw0Q2jXVjjL0beidR2Aqc73thliIVvyFt0FCvl4F
0m0CYio2JNk70si6RxJnTM5NzFocCCukRHaYMItoiX99osUDG4Lp/uSVCsnFBWLuWGO8a+zbh9odyxy3xGvLGLEgXdC4ax+4dtzITselcHZ/M9EfTtdxnYUz
6i4WY6fndF2323WGw4FcyMlo0uuO3UFHLtqt1rg/mHSGXZBwC+WMZHsyURPZn3SBmlzVkWrSdSf9njPJRn/s7p8DPHZ/p72/RqctjuDvhGI4vDWh4XbrubiD
ghqrcEmktJA0zfdIxTHtzx1lS4SqpT4gspOSoAv/rNWmwH+O0jbbquuYYjwu7PXB0XwFskBbscZfWtMDN8/1kwdHAv7nqoWmvxrwYWCdX355cSXDZWQK4P+8
BY65pj+IfxYfK8a9NUO+WmmI9AWSD77QvNf6OfPcyo3VLP4vlChXc5OsVV5Z/vi19qMmzJw9FsCVtBXVqtTTRkMVb0Mft19VWKu39NySWZG1L2wIPQdF5NqA
5yX+Y6bOuyqRmOZg2ZLRjHU76OEBQ8iFUrqNn3568ewbtEhrWomaXahr2vVqILgl6ANTRJEW/uk3EuNp+g3wO6UbNJNNWv0G0AMU6+/Uda2iNf2nCYTB7iR2
oaa69NMnp0+fPHuOZhRFIsx8EGjTCkMzquhOeNOktA+Do60nP735dvbT6fPXs5evnj3/HjSp7WqlZ1Po9vT5m9kPP33/fZ2po98YiqNeDwOd4PlWoNOYeMFw
1zAd1zN488ZbK4aq2UzkjUQ9E/bs76tmjdsBy+HCVE/pAHVdQwWILDuJgTYVoVxrhn4P1ZLkVcth9sbI+CnpJtRwvaVxqgXCAthmbXMxpYY2F/UWGEigddUO
Jy2kpzq0BOZarZ7tHAaZ9E8GgX9dWyoYbBzqGZLPpC7+NBXULzmt6F1Dv/D8pIkW2ObrqFav56b4hxCwoVxajfw6IPESeRAuvYQZabRhmkFTGjRIotm3hhx9
79etepp8q7G3aPrW5lBZEJ8BlhO9bKnuzMBbW9WmcKV+lnbO20B2xy/wTWlvFrbiI4cK8mPSte7LdA0KGChSsOIfsNcy8VrYM0ylV+HTA19dsbKDYzUSkSVu
+8D/tVqgO3ZAG52oIcnY+XDRmbvSbXc6qqcGctjpjEaD3kKRqD121eUx0mBWsBaHhpyj3WhjaGS310W5elSpVDSva2o3j4lgAvXKeJW1atgo34hL9t9IFy0x
gdCJgSjXEk/MzljWvDxX2xB9Q/OHppqOZgM5LldX8hrj29aK5TNvJukt3yQML299sC9IV/AST1WLZvzgSIt2REn4kjyHKvkJ+EmKK1idWpnYhiso2yJlLtFH
whVt5uHMD9A4flTh2qM4wWfK9xCieR2k2Ip27gRh0spPr7//2bzMVkiVFtLcpd5tzxZpMRGZkk9hccF8+2u22BbWOSkSA6f/LfBVMstCE4RBDfYM/iUx0hq2
SDIPbwAG2NDLJ/9t9ub5f3sDZD1ozwDrHxw9+eHJ9/96+uJ09vPz16cvXv0Anyoay6LmZQfW7emrH54+//HNKXz5yHyt4oYwssoJKE78q8GvwG4kfkBmBf5a
rK6XPqCu+a3Cyk1Dt7EMVgtqgn40+IUDeoj92zcPczBKImp9ic4ZalB6IeBp2uIcKBD9uXNqNn3Cst4KkODC+kmsypSBL0krdgwANWRsHawQoiufek+ibPDB
AVIMZforIiYMMggoOZS6yDnQZdoNyGyX/KjUR/rU0J8u5AZ/rxFTdJeOCgmSPoWuJL+soQNigFCIedjmoWE+xDGDbc6r5EfeRazsidNmn54z/27o1wyuDZh6
6Bbj1wAVP76AlmlVvEU8l6GbNgdA5rboB3Yc0Ohdb72WPLVwGfj8mIIFHRIMEvrV0K80Eq3ebyMGBfA0g1nR1gdWRpOKgtV2s1nCwlgDCZKl1D+p80uy5fVv
3RJwAyAxpX9KBpPHewTmQdnvLfChI40YKvWUPiGsAGZB5o15YJRMnzSg0tIaM28eHJ2+efXj7G+vXj9DQkT7pSLRTY2sOaDAS9pfY9dOHHBIhXONXB/FaoRO
U7Z5ccNG+QI0YiEvQXAIT5AvYY272+jpCEWETAnKuORzYycAiqOw0iLWC7oT7zOinukH4Rq9CKpGmtYJRsLWRfMx/qs1LG3FVCqt94Hn1+Y00DkO2mL5rbSh
yg/ffPesolW31lxGahGAzluvG3XVrgZiyvEQF2pze1gYpxIVhqTit/D7LDuujxTTgmOiHxS7Doqo7wLF1sLK2//xy99mZ0eVRn6qPJ6OeCRWyq9h3bp4NBXj
dhoog4OF9tLFu0lHaNjsLYOknQfUxKwZ1c34yR0LjBm1LJoB/mgIuSL3LNr4uCUjDA83qi+OW7f7z5nCN9ywh375VkT7WwCBX5zaZhudvxXNsz97/t/dYLW6
Ng+e/1YGZxe4APVfnDIgndjmLA4YLYxaBVucAWer7+oyQn8Mk+zf8XfT+r26/juwyWUk1/HdesXKM80H6hks4FLp8lBI0W9Kq8k1NF/SNXK9eVy+Pliubqww
jjdBggU+qdxasuRUShfD7UMUu173spJZmI8Jl8nyHix4I5BTicql56ogX41fEmdRJIhjqwa0uFT5GsSiEl61xFgJ4rPeiqQX1zEtbH2KIanw8HEdtqi2whSi
wgy03MtqCBnxnggEs6DEyEH+yR0dGqqtmKCvmQ6wB86bV2egKYXQALDPVqCfrqBMJQkww54Q+vAS/2lk7cGKmRl8Nj9JXLjcjBkVwlcuI3hlFjzf0FqhsuBF
ayz0FmmQaJV+eH6KJwA68+5jQh8IDQtpGzkVxUhIlmTmFetlN2cWu5lhcLZrTLAaOlYbIrW9DcFoDmvMay5WASBnNomgD/bv2GRGtnNe365Vvg+WS4lzWvyy
bbcXQzDwExM5dQxphbbEpZCOUQeYT3FQOVeBt0jaAiH3A8z+gJF9owekML4mFhco42jfSvCHzcaXUSvHJ3Q3FmDpmANRdBaGXoS74bhtUOP9GuIeyKQD3k1v
wQtvU6N3KEKoDfFYJHo6vK/88qHdrtCZJ+RA+UkVXBXf8eELrKB8Y1mibO/8su22O71BG21iaGwONJqfG21aiy/EMwVml+I4Bzo8heM8BjN7RZE1LH0Tnqv3
D3wyYBUf+2hZ8OEdhRoYbjkAwZuTgq+zUtkFRCiewhDhhS0CuNBLLkj1ua4BDwY9AEQyGNuplpH2vAdyT3zcBPbFEjDPW4pv37z58bS5QkxQff9CbGOKClgB
NqOwwxMxqPWqMAGibSTy8TuFrhI671A5q9NwuSQYtOTVMbas9Ql9Xfi1hUhOYl2HvCFOc+gdfsUNUtZW0q8HIAfPZrVKpqU0WapByJjv+b465zerldw/Yb1i
MHibGoAX1vCPtdj4WEaXXP/t2U6yoUDnGqJ0Q8TbzQp1LkMs2AWt/oFLe85EvYxiqAP6r3cBOnBmNlqbMuOheDu5JNqDzope0+xIs/TdQVVQj9PQOamHw/5e
XIXi9cN2FX6W4Xtl5iFAYopzaagcNxKzFJ6uRMJ1tHLK8873qXWmDUajZ+awV2mahYr8bjVmPPW8jvSxgkh7UhFHoqB0s/7EFVvEiY5ERWBR886LVyr/Eiah
ovpNpp+/Y0da16K+CiI3UcSs7soagYlTA6l6ietjY4ZuIIldW6JpRvpAAxCpng40WwKpe5YW011/IV7KDzo+DnBniTYcOgIjdsjyYVAOyW7SZq0eK8aoEUMF
S2mLAd2oRQMGhGsMhsiuQOogTzzj+iOgByIsK9BsyehxJVDCBtEqqrd4q8SI39TPlAhwDNPjwxuq9jb9riE+5X+0jTfVfWZGpXVmHlz9rCEcjJua4c75dNBu
a1vvK8vt1uLYPkZFdH8a5T2v7DREos43xJfwhJg1rVQaevd7BiyNHgm76Beu1RT5l8HpVHW5TbPi8paasJPd4TCyIo5eIZvrDNuGtVg1aHzZGkwQUKMPYvF2
1vh664ReRgygB8tmltgzMxP2D/j4gBIBXvRRmUh4TAo8hhfCRgv/9FO9YcsIrmhoA/Hzyy9LLK86utwSkjmh1jO42dBHydxkf5M8oAk2AtHNgtDgo14co1qm
J3hox2SqfdSt6Fx2B0MaQAtMCNB5gP+1ztUH11sq1D0tRV/vhUZTsEtoIxJGqZXn7BbYiT6OXMk94ULjI+FAcWdKoHfLwBDK2bCu0JpXToRGh4qBJ7wyP28s
RZkhZeFGnm3zF1ACF4b07S3JcpWjpOCUyhSK0F4Y74zOkg2l3C5SunEKmlNG5OTWO5U9uMQ76V3vzKR2zl3JV/e+A7fQTkm6SGqn+GXNbjbzomT39jYmhrvp
B0wK9xR1vOV950cnwYh8PlbMVi3jZIJfjQwKGrOXfMoc8VxJ3UfooTQjEk3T+u3sKNlpRVazDaNNuAUxd0GqN2in0UXoXcpI22eXlvamLZ1y6uegvBmIl5kO
oah/4qrpaTKoSKPgyRYJgjSZqWb/utRbrmcwG8XehcIAPvnBW29JQ6nVEvADAwb+V0tWARl7PbeJD9XLR1Euacw4oNpZVnzYX1CK6CEdvlf+Bma20HZGDFY1
/wwT61+w7TEA7SajnJIA0V4HQ+MElMyIUshbqLgX/mk5Sxwl62C1YjeeYHR507sBmtZEhSUIVgcq899wNVLjSQC/l8B/VxRJ/D4DJz0t0w9MqqTvglhFh2Ut
p4nmhK+JLsqJX1tDztV4a8nkM8JxlOsm8NvWhPcqv/UMCbesUZufme8kPoxSVSq4MpLiFg6rafwPlBxlDCn1bu3mP0m4l8WGpmZfGOyeK9A/rACj3CdrK4TN
AT3DqGSKoAHT+U2j6NK/yTEWjuTiYy70hY7LTvvt+wKE6uPqyg+1DuZ38msYyUKvYUod5G67VGQaZ5Zj0StSkQ9SeE+ZDxkNdrfaW+YnU35MaWKm+1b1HnIk
AbY1/rQv/cu0bwrn2qB9BHIds7u+IdKNANwyaKQu65tDujHEN5vRyuOftDMCuvEJWK3R+xmZ8uhJscx9tvUtw47tfV69+tuTXvvsLN+OMdOpKdugt7YQE1Oe
Wzo7YGZkx85mZGnCgvhTe9RHub7rLQrwQkLMcnJdYHbugdZPIR3A7rBlJB7qaPrXXV0ljTfwbFns+fOYVzOn+eOQ7tUBzWVf6/UWhsCHM+e6VmnaU6EYrqRf
ekrkSi68Cx8xas+YC5Hai1dWh4e1yP6qkzuu6LTEa8OS6S38Pau/bZ/lvFjoX6yZFjEHS5iPW6y3DCBrgKrEqc4sDqvjsdTMSnxQwmdpaVKvA/PRniGfdHK3
Mmzm1poXcjPIRTVDNYx0kJvmW+AMuNcVoYaFQZqpEboh+f72pNNtt88auR2rTc53ZX0k+Z23Vs122GanVww7xswOvI2mkzzYwEujGKkd/nlDRL/hrQ2C1Vm6
BPpAKJ2rtTZrD9yRsXdfQIzMDpIk1laLLTfa9g4Lq7+DWCyN1mtJE7MNgBvftA0QVmp/Xtf/R7PZ/PH1q5c/voEftT+f/BL++Rf/7/9bvcJuq5Q4cNTUxqF+
8NNNTsh121q/keHekRoMYisG+2RnHf4iLyiO4uwQjxwGs83SQ0UlyJ1+/ASl633gQNlccFzeHtTh5LUKGJmVxCLM9E9Jjmaz5HxXopztcj7tdDsCwTVwWK2M
qpp6WDLdlmyWFIpBZ9DcHq+OrsHEg9vzaV29x4bZYDCen13zvLONu8fblAolHicDUOELDinAICvzNULvNG3pc9NqJvFcHX06x3i2m12q/ccvv8y9bGQGeGKi
DYmjJHyLAEicC9cl85pfJJ5CnZ0CCuBHPkpntZicikNO9fHiRFyyGd6AHwAt6icpYsfNXJAKYwB3Y7cJ3VG8hjvbRq7ueBvhqVnif9nPNtuspNkDZzRSqGx2
4KERDFXUR3YAsh9vGFUrXJJe2G3hAV80KCjYAIFT5lzHoVHBSEfqz0gIVjYYkbYAsmCF4e3J+OxMN35zs9MM2+Ga2xPYbZI5FGKnzYfPE9SNuXK7PaffdQcD
BU1M+kPKueu0O92Os+i0J3PXmbudA4K6k4FZId2dwdiEdL85p7xYFF/QzB6o/elFeo5On64sT97VokBpKybYRPviwdnCASsUgemJKegZEK28UAvH08JjgWEm
qhlziPgzkzSyGAidHPHyaAf4Xie87hosjbvlptC38Lvf7mcLUIap+TZOeiTXPjF29PBD+UaSdqKhD5pnW+CVTMGR6VNDY4bJENLga/M1G+Av0nwM5VHa+4Kz
TY00Z6KulB6SMOvy4OjH189/fvH8b7PTJ99j9HbVynlGCUooLdNlp8oS2D6b9A2gTo3wBw9ErZMdWfZK8oengJR8rIhyWEyrXLXa4MAP4wAA9XLGu5JTrSOV
7lOkxovnwtpMuQ90CwLqyxoytWj6sYqH16snYtwQVTwDtQjmW3ymM017m61uViCIz4MVLCxUqFpqmJtsyLKSg37HVqtVvTGaMG8h75w3b4ZVG0ki1bKpD9uF
7a6kxZ9ef59pMAm4kKJGASX1/Y1jKItunby9u4f6JNmG298iOosbhy1FLwUTb97t6Pr50pdp/ISo0dEQQHEKM7oI1mt5yyw7Pb19a86qUbYJ1i3pYJmlxLDE
mvIJNSoHMpKcj1UsXz0reGU5arjomkj1Yp5XXjuuloZOWUFr1eLBTOrLnonm5DTCwly06lOYSguzCtiH3vT8YBWq6Fd9a0dtWLEHdjFtQVQb1XouzuOsMGp2
qSYxG4qOL2p9qcThqWsV+Gwtc9Sk3JepW20lCvSte2E0JJ3ihRgXpjuam7FQehXpujPK1lFDHRkYyENR5dByKNkyopDgYFl88K1w+KfOEaaoJWEnZjsuLxi/
KvJktjltaZBCMJmsBURKMDO1mbIBDaXdgUWl4ALNWSgBFxTv4hmdJDyay4NKeh6AqQFiAKtWLQTTqSymwm6blMUqfwK6rKLiVs1GBZl6KFk4V0U1RziwxCVb
QBSbUdqdMcepBPZarReraxOWyuhAvpzlXaRdrYu0vpbuKfykszC1krZNgqhpUgPzEETcGQg2uYqntlRlBgV4Mx0mLLjsDKq6eltF+wII80/pxCl/4+aC4vNM
KY0zXBAxrAwtPnGS2oGQHDi0nSDJQJhPllSnQyJFa722a3AFi/9kt6A2Q7LtYRtaWSppUHslQzTE3Iq28zn8TolsUf2Y+j9uUjfGOR5gobOKUUt8rRaeH6+8
pRTu1lmxwHLSAMeIwzqDNW0sE8zLEDWJdmO1slZl0+AkywCq2TlNM+udbZT1ZhCdJQvdEN+pa+JODfHmeqP0zzznKoF9kcd8vKmXF2th2Dy2WNtVoMBkTbQ0
psWIlog1Zg8FQa69RnEMENU7l1Y8tT40GkpYE0vMImbqsCJuEN6i26lsDRKe2Dc8MYlRyU3oINaV5XqaVqo7EHovnVHvZZpJfUdjNmu0+JO7XW+i2kfmLid5
1tIQCUM5KXKTPRozD+ZE7BzmTSk3xM7wohV7k2knWaCNlVJmNWs4J5kOgT4+VpNhW1NMlYM906A1gEr0b0PkJMxJFqw39X0rDvyYpeBeJlzQD3/Wyvx7wX5U
rSEyspdQhxQL+CJLsXlm+wzvyiu//JIWM6Nr1u/EPLWFZ/zTfc0y3Qx1+vwNCO9PVcvDyCpT2gJwWB+zH7vSTqSINK0uQbXcKrR+kHUuFaZ5NJ5m6PPiUob/
qfw2t6Rov8OXEiwoUUAP50iZbbDyglmelO78JJiQ3QHahwy8F2RK/OW5VsB+ZaUrR1G4uV8oCi91YWvTv1Aq+QQGBrLMTlWHBBRK4ltu0OwMHgj9FA5vi2ad
YfIUEJ0D8AGCb6fQ27vUxZU7jOfZCVkPYHtYnaYOX/Uva+GqODLNwaEob3lGVkMAeWTiNNYp/3M/S4ZT1ZYbMiWGYRJzt9uE1NaLzjQ+BUhocWSFFcGcyC9j
veVA6GrqYUm/2YGvVXKSpN84/rVAqWwinwhESrYTy2Lyq0lYUpXDkm7uYrk1zCSn+t9PNOXuqdYYxLMDR/eq3XYI6SdLFWTs877vMp/3ZU66qFQ+WIGBgXGx
6DNJLqju95EKjLz7hII13WkeRf8QSXFf9mGljD6Ie3C1hA528I5P4BFfZd3iFsswYuFgpnEbr/gEDE4Co4r+CPMpEV75c0yfoml8OhS1T+xgIGIq1g1oY0li
acwSLxeKb6dbcOYz8eKZPhpEmXk49SQlIcUmPo87L7douWjKg9YtCSo9XFl1gjDGM7ugfIY6WkE4nrPygP9f4PUDp8xvrFgCAMCGiwIsLihTdaxa/z9DAzIA
rOCIHejwPnAsXCguZGHbCRaT6xTCGXYtaFa6mI3dWd5YyUdz7FtwPeq7qYQJVrArIsWJcpdv0UP/6TbEIYiJ1o59ppYyqljI5xck4YGYdx+hhgESpXvv+y9A
SHe895f7PDvzQ7czGUw6o25nMFJy4Lpq1J20FyO30xm1e+1JbzCQfTns7duZv2Wc1kZ9b0T79MdfilNOuZYkgn7Ol1zxrbN0v6JJ+JRmz7cTkHP7UUt8efzg
qJXuw2IGjI/oQ25eeW58jonqh+3Nh4cY3mxetR+Km0Kl8w7UY89kE9cUcy+KZqvdG6h1obyvVthLpsm1DGEKJ6Lb3XzA541O4IhpBdebWme8+dAQvcsr+AMD
qj9E7uniVakdKB8FmOQe7PBas3nZa+L5/ob4Qg3VRA2Tok3AWG8bYQJ5nBFmMF+GZIalNaNtuCBl5ApvbqyXjvy82xCFdz0CG06hGQebIpDoloOP5gqGE/Sp
wBiWEoqOcTT43LwK8Rn/pvBow/91acAJRAb54X+x6Cz6i3Zxol0sqWG88JILex9mVrjd/qeywUoYbtIltiQ6Y4MJ5wqTdZyIPg0smZTnI+SbPDfgiUu/SSFH
sIjQqwoLA6QG0Xhq6jgGypHtA6vBHOWrILSWZr3FnCTii+Fw1B5O6qVDfitDTzbnnDd7ioh5BrPIt8TW2hedQWfSIfSwQEnrjuP80IzOpRtc4QogisFMxRea
7hewXgDJ5pUGw3CQX26ytqzlXoaeq5ebF+U2eqLT0ydgCdbeYqbSKaWLhlFh1GmuaKw3wgsfOD4QhpFZ6TzZ5RABJ4Yrea5A0dTzxMPKeKNWBgt5HvenwnEe
izXoi7AwE4SZAFOjsVzihRRzuXooCOMMPnZaw7IGaMffXg66yCC/iO2ECzWdII6Dtaa0Ynt890IB3i19J8PHEhqhyRFnVCc6orus5RZfZIFROUW8paSBANFe
Z+D0B/U8Dxgtxoth2YroJSjr78SLarKRTIiwrqERp2FhVrRdY9bk+gkFuDT1rRMwxmAb4wqciF7aHY/vofnWDBaLSMVUJDcG9mSXMMYCO7RIp5SxGHY5ZulR
2osmyoSz7iiWLuMdGFtKFnh/OpPKHZhdZiCsLUXWUFFiFcZrium1QYTZhhFizCbwNFQKeJgd564mWwGFw0m8ahZPNueEGwvQTM2Ir4uA3iTd21NY0QzUHH1r
9kPxfgvKyeLaSKYTQSTSdFR8pfCqGl73fqmIzHJc8ifuELE7GsiMib4q37VUkTIQ6X4wedrlCSVvaoKxuHKhZ2wBaokOCOx2AUJ2xaRwB/+vd5uGpaumL4w0
uAXMBjlTNaOIEBke2G/nu0bBVSLI4G/T3LoFa7farlHnDMmap2TgMOaYDsXhaQ481YHSBXU3VN4aorMI63WzOOMCrKjXEo0xS7k6ahAzdu7gIGyNEJ3xIIsV
y5iRjes4mYRs+FU3WVzdEJqB4YYkrc3mQQVvMqKZO3w09oEhfQXy9RAlJy/eBgW9Z7lLrxzeTjRUG0br27oe8ppxCf8aJuii9QHixnoGCdPPyqT2oq8We+af
GU1y4e/d4XibGpMD48hmhDt5dal4zYyY76/6yPTAGIY0fSyaHc3qiR5LxAQSn4VIOS2A3AS4hZlqAV+MZXfSaz+kY17NKL5eKSM8kkaZLr6ie/NEzYLJkMgO
GtvHL/eStqbkNpMu9CN2834yhnYveUFtKtB5ujbMGmBWexwBZRccFqzrskKfxwUwGQ46w0WvPxyryXw07IxG3V5/7nYGi+FgOOnK9kSO+u3+5AAXQOkg7UD9
Htn/LuhgdIUPgOk5bk5/DyiBXppadQ4WPkb+yujan4sabV3XxfQxH0OhZPzm5rApb2y3QMNbqrg1XwURZoOpvsUtkybGcuqw7LNq/SHW9hai9ieuXNcOn4dp
q5zuaSqSwUGjz/n+0a+vX7g1rtjCxiPsDtpnv1PdaiTZadClk0GR/7/e+pVv+1jRHT6ZoXJNM9QY1SGdEeRKAs7om8qCEJr0Nk4AnL+Ft7QqjCKu0dj5BAnX
N3setLH3lFUTDFhPPGW//yMUF8EGY4HcVpXq3Ig5ZnYy/XKbpC7XgF74kRXr2v5O3nA4EXYBguhCcUADd0bByTFmJvBjYlkmcCOGlhfooougiH9txvTg6AY6
KyOeXbeSp0i5q4S5Bai7mHfb3c541HHmUrbdhVKjsZK9+aA/HLbnnf5kMRz23VYLhP6gI3t9d7CYy3mnN5b9RW/eHalxW3U6vf6gPeg7qtsuvQVo5ygyBLSz
FFLPuNETR+NGZ1i48oRvtH2D17jV8O9TSclArUweq0WLry3EAKLauYzoNBOfj2iIirXZUKknz8mJGL5cwIM1wZSd/tJc8pJr+/mvW7kyN6/YR/JnuLVLWZ9I
D68lF1bUW7hlYjIP8XmNhrlj5ak53PdpfeXPDuMnsivoHV6fgjNGbKW4+kq9eGxD3/5bALAVaB7/tClEmOsjJPwPprDCSwXSz7vOBtlHaWgXnGZp7WOmUHgR
/RDEuOFdy3ZCCbNneFjQuLBbOU97PXGY5889Wets7/vuO3qTG5s5kDPdX4xuE5vmpllL77HhCCf6VjP5N6d6d7VJB3Xs8VGTAV2/euc2qdqOJgu3JeVz4+os
ZpwMN5mYvmul+pdgtSi0abYE79AmDdE0+gofqplDG9YFQ+wXLLtfyFxgZS1SJszNGr09dwp4zLVq962vhdW30M1I6eUNEQlYmNAfKE7+DDeSMT18nlYSpRkk
hxDPQsqvShulAhNUC75zovVL+IuP/32n1IZiNSO+Zg9vABa/+NXiLpl1GBinRHAyndVLiYpYpK5WL0ftzH74Pqgle0AMwGn1iUBtLbnUWCfLrhZTCoWK7mfm
TT3XqZWPldlg5myx5nRpGr0DaurAn9sHd2vXmK2vIWYz5hOzWa2q8/eB2qMz+Jmy5Vn8iphFNMK7saEi+khxytyZXMCn+xyOLNtgTZGm+mK91pcqmu5Lli2z
BojMT/V13I46x5QwfHsPMVoLWSn234Lta4x5jWq3RJmXnP09VEqWZsujq8JySLRj1rklcre8J6d4mbxo5rl4nSPqgsQE3AAYgx8AV/bdgJhAVFiyHWA/lXi/
L+ZEqX52anwOFUsal0vp+eWsY99wGOZ0wm4X/7CXhXrJjuv24npVdgz8Lks/pxw26MQqricm6aeNCcKC2Vz6M0dhjttZHMziq0AnfPCQ2/PN8bOVkhdljN0m
INMqQT+jXLBAtNZ+p3jSgtCWoxyQv6/1re8WGs4G9e5DGw7qTTvYC3E8BkMdasiWFASdjctSubftM+olnUxxPeQKGKAOBaSIrRn5xiN4gGrz9Bq55PzyfRhi
Pif/raT5CpN/6I82UPjWs6l4m+VXK7l2XHlyhwW2V6H6gnLTkEBqHNJwyQKXt31Qc1ZC073DTGn5kFYLnOvWZg3ruiM0crFgd5/CWYZaaYkp4RcudU4updIs
2jpoLtWw1BT/1BslYi6Pd2WnqeaU6LpAGJpoZknUzYxvgAR2NdN3goGxh2SR5Ki5B2G8Ipb5DGQYki4Gonxe1vUDXqmOCq5NRDt0guJY8uC6l2i0x1OW0s6a
6oGqKctBU8e0WVxCk+5M0ytartEsuoKBQxd0dcoMLQBSI2BpVlsXpI45tnyoEvEcT8bQXXPCDTnCGe+/EXwRXprBViS34QhPWJeitQo8n9t9Isi319xuhLnj
DalCioiutG/9AWIouUtPEHBoxHslUkbKHKRq5ALy31az1/ZUzwBnk3t7dmplZSznNsVs36g+EW42tOhIQQETcUndGelZYGnQ/SkJpiV6Tqgw4wVoOokRcigO
PjFXshQF5ueEmx65u0N/vW0Ud1RiTW+HLCN6q3yVVrFyDxQXgxiXTigLy6DXG5kA7vZ9luUw9zmK5MbHz2hh7M2aog+NaI8AyLa1COZKYkCj8lVVX3DwtvpU
QtPoKXqCNLeLL39uSjFwSejkM3MwmipPcmpP0T7xdtsU32LOxDQFIY0MQPQ2WYCzP8KlshNp8vZTmjxxZk7JIZ6jU5ZCXotaCN6HQhfAoNJSG5cKdULcRdXy
jlm+MfHRu2mJqjjCUdJ9JCAo56FHqgx++JKS+e5A8MOVldPnT18/f7ND+KgPsJwrnTfTar40J+c+VLHZdZqc8xa7y/QOeNvb49tDVRKLY3hJcjwcr1jBhJuZ
m2mSBncaci/8moYIJ2iq3VIlGYDuHLNtYlqZqaiW5t6s7hxPAeXoZBqeOgjIcfcbNETWO2Fdck4PJZpxpJXh4Sf4zSgZDrq9YKRvq3i+T6Cvlk76fUD8G7Tb
7Q48ONLFi9QY8rcaD8QmOSNqt31WakEckI4hJSArbacZNWaNokG/l5eSieZErqD9WgezJ1Xp1rCT42NE1hNz29dXejUQb/WG9wGzwcts+OKVT5xG9Tl3X83c
k5PeXXZvf1C7iFw6VfUsyULtsTdvIZ2Q8+odLnXjcwyuICWZcodV76SJ0nFm1D/1kO5Wm6afr56bK12VRPmbolngr65n5qL2mb51HSgMSepCqQ3qJUuMDShP
lxXKK97P0DttwNF+8e18sr/4+kt8FQhRPVSkZzJeQCcop3O9VNP9PW77rH6v1qs8VtH5xX+F7erH7i/+m6uAFIT9JUrUbCu3KkkdBiSs1aXyMTdUotvJ0PFi
HNIsTf5ZhPE2s49G4Qevt35+Ky09ZGWLmobg64HsfTsK4QSO8/FG70RPqzhsG9E4n63uc1daW90xjG8K/+nj+VVKAl8lRRF/htPqubdcRrRXXc0pjZRSdFr9
CfdJ9TGnqlExQXGCf7xVqshT1h4NpelHjeonoqqzvbhNfoOiVWeUNQn/Nh5mT8WydDaRIlGqN7nBUOpUaDeTNxXrtFuDanqYlfKi4GVkbnFzZJePJH9ubZ9e
oFPO3l1zy4PrgCZyuYcP7jrlN2kL1TPOVGUy4RITyqzsXhXDWhhWM/K3h9xOZmbvzLjMcgD/jC6x+5Fk4kizSVJrof+VNGnIMFJQ2yZA/eKzu+zuTxNlkkxf
7aXdoqgaUsDYbnP5D9g5TfLQAJO4PsTJeYsitEeLyd/lQnua+uqWDEuYmp3ZEhVAWytQEiCemm/ki5jpKx20jNoASfOODKCWG91m1w1vt+vIpANbbh+Yftiu
/wrD8ABU3XrpZaGf1QC7ze7iLtnq2hmkp130W68YnWd/+jzRrIuOch136Iw7yun1+2On21uMe91Bvzfq9GSv2+86i1Fn5OyLZi0OzY5h7XQpiPWuQVx2eezA
lHu68gCNGsJEl2VLot2W5kRG806V5zh+avSalO02Dkp8nMulbHJNp16BYlTcTy/uGhD3CWFfemBNysf2mcK+TJuBCafaEfj1WSO+7hvqlR0bYUuL934pyUFK
y1bZYlYaLkU5kjUe7T6Kz5FeUyvhCahB1ZM8q7jJuH8okcBzIzHKescULbV87OFPL0QI7DQffJgiFIwZNRcd4oaw2Rvilp8cusW40oGzSlu/KYl1k5EJdTGp
AiKMjKBzFDOKkSYTMY2J2yltTZiQiXnDXapsgFs+sC3L3zUmbILIOKYxFdNHk2H2ROfWaySZB/GfG9zvw0sgb3Wz6punWbmfYbhWA3Mn7auyK8yoEFqU88Ps
6lvrfm85eP7MOOw4zh39FVaq3lw7yAQlCDzdFDovrVyBJYJfIz8tHh6rimaKXLyz1DdXaugfGkRZfcTOpcfsXKp8qNQfHetX2YTWJWuMQDRLvNNYKE75wC73
gu2fV/FDrvPPy/jhwdXoGDvnKa98bTLCVKple8AcvkKalkS9hpcgChdlGhXZL5+Di6WO7Ztb3SPWKlhDqGdpo9/uH9oQkazVUiNNx4ao7niuq/zqzf4OQN+Y
I56w+lBLwoEAcDM6XVncSaEqt4qO4tC5XoHRHDLk3l35xT5fJB/u5f1BbdXwJqA6cAvwUFp9VRLlyUhn5k8SqZpPDlOaAeYOOJEDLIf6Va28fNWX1yI6D+Jq
jo/vY9P32ALb4UmxryBtiNS4qyZhidX6rYE6aZQOmk24aUGRvLf5kQ/fpCrY5miIoQZA1/kW1qyaS6BmvdK71NVSy23vQmbQhLWPMtywF1vGsZxf3Eb1t/Gn
u+NnaX9l7gTjHMaloD0A9ObP8IKdmU5WDzITI/JnlBp4t0v8GzxznveGn1LaiIwn3Eodf5i+Y7JHpyoP9Hmz321HNT5Zy8lwrVt7S/QazqV8xj58DZYq4zAD
w3bdmzzVxVYyGZjP7q8vmjsHSrI6czbnP0Z5PFUJMsel0LVvxqpmeFLVviSLyKkcjjc7zKhPAca9caR7C4EZd20s1xuFV5FRf3yiBrkeG7NFbvnJ1FLdQ5kZ
EN4dFQ9nmXdbBYx5+FDCNG1UY6Dd16g/RclB3wQ6uBQmZjJ5GIv6wb2V0swg76KV2orDXbB3H8Q+WVMzIQl6+myag2ZP48MjJSif1okrFURK2QGCT7R3qyV0
fxuT+sxwMEoFEa+J7jNpJ5OZ/CH6aqnW87mUnIKqsVvHKSDa4BBenNFfrFHv0qKSIieYVPbmP7eLO2nYFAaka1ih0Xcbrlm0HSC/h8JYT5m4yYF+tjtWludo
EjPaG460XTmjswEzs6u2Y+f+P3XL/o/YF7xte77ssODtPC2z0LlMtFWTvnWaXMD6X+HOu8P2eNE1VAqeHI7xTlsSE8pRN3iwXSfd/zxs09p14wW3pnDXHcad
EbqHDuZHc4hgA/Z9cCeXIHKnX5Et0cYd8vb0kogTviGCo6cwJoQmenPoYu0D0W3ux8KMyjYIcX8rsy2oX+h9v5Ejhz2pepNJX47lZDTs9bqdkTvuLCaTgdvD
Pb3+qNeTrZYaT9qjxagte4PRot1v99R43IXn0dAdtds9d+hMuq5Sg9IcHabTzC6geYl7f91GpyuO4C8msN2Zv4GvNDVbaN8HS1BufoZXD8Suq1X/JQr81yra
BH6ksqXsjT/Pp8MvGOcTnz8QD5rZlAzSjfha3AZl016t8PImUCz1O8qmZB7on/x1qndpwL6G1zSnU0QEPuWloSA/bhgv03rhIwgQ5ePtxhQ1bl94iRIkyl+6
at4jh4/ipAewh1BxlMngkqwU6QeAj+AVpKDX9DAiAq9W0dHqx5XsVHLboFr7r2TfJpdFZ5t6tN167kkiiR/vapv9MLmm+eWhLRvC3tWF+Z7rxLw+tButX+zq
RX/OdaLf7ugDF7CZCg7dJ0uznRDLib9ch7mv2LFIO7Yx+RgMOOjDwpIGt4ShrdbbfU0AkRw/gjcn0qUh88LhyBPyaWVXGAqmq7uvWTknI8K0z08GLmnr5gNb
PFYv2Q/YW4bVmmRo0TGmraTrLIDLlbzVTFeBmSv7A8wEvhgNR+5oMpx0hpO5GruyI4eLnuN0HKc/brXajuy7vbEETjyZjOFnV84HsjuXPWDDbmc0GLuTsVTj
LNMt65k5b9kXYr/dxhDYb7dB+cMBkI9Wnn8BknE1rVAmuehcqbgizkEHn1Y+/pPgfGRgB0fRMbEvKyF5VfzTTeXx/VoxgQOf2lqSJbdpEpmnTR3dramSlOvW
mHjvDjUr0FWjcJ6p/j46pnxImDG89V5XS/YId9Q3EfFz14c6oHJ7l2HLV/Gxv1kfw4p9aAXh8qtuq93qtI9dsBD45drD4hUSMEvgRdcwtXPZG/eb3w5Ow/ni
evnt+sn2zbP/vjl/+e2vX//L9/PeN+dR9N37v4yujp+qp5tvXqvTxdXXf3v25rvNxW8//jieH82/j747+rki5mEQRaxOAj34gX+9DrZRZiqPjjG53+NdhIFy
B5OPAQ3qTGuo3OIqREV6ua2wJqNhv9PDZPoTp6uGbnfeHnfaUo3GvX5nPm/3J4O502+Ph0BGw8lgOOp2u05nMHBHo76S88l4MpLO2O1POv2x7LRH/fZkuIuM
bh1QnrpurcA6zwhVHia5R5LzjgH+fMRklBczDgi6uRGAUHgrK2ZCxMMs58FaVekGVvNGOxoBu7yIkpReKqijfFgLxLi7NJBJj17BAhWrqZRA0BtnfHDcXvnF
DXiULKWVy6WZpNxsmgCZpgdkWiEt5+vgw7TCuezh/yuCRnJO+57TSgxKP6AbsnfhTisvQVkcXHb6L0Hb7J53+oiJ+A0R8nL5+BEmLH18eiE3El7gb8RP+fhB
83Aos05m/IoZaNGnklczELYHL8IntH+fNcp1d8tiHf0nwMl+nfPiln3Te2x/OHjvPaw/flU+CwlhRJWAYvAZJMSgwqn/p5XOEMQfpb+F3/AxhDJdpCusYFHe
SHQGq16r2+y1eqLbGsF/Q/iv0+y2uqKLv16OWwMxOW+1O6VkSY6iLFnegfnZql12QaQvV9e/qRlGdslV7hseEKXPByPQ5+joPiiR7fcWjNghbLV9bUKu8OaL
opDdVejzRAm7/ZEzkMPRcD6U/fFEdkfOqD8aj7vOXA0kPDsDF1TZRWmU8AGDTOOFKVb4ERCCQaHMjR+aCAqRUo8B5T6ZJDXOmMugQhUFK/TOrjHNK7kaaYOg
hB/dhhuPv/ZWLiiEeN8S+YVCJJRdY87zqc816BL+d4dqh7LGx+wKTO41o5mi2vT4wS/ihwAT563o6qIYq4hgQdh5C+4nHH0n4mdLaAR3wAgbdfsjMKrGiyH8
6QzloNN2Rgu5kOO2HA6cdr+7mLdaYzC/+m056XfbXVAeO4NFezFRC2fRmQx7PamghV530h/sUinLR7ED/3OlSi02EA54b4AmAramMFV35THAekF3jkWKDhBj
O/QAFu9qBSvAGIGXz9Lrlvqw8ShBRyxMPfvdYyh6paux4yy515EHyjXoVkcuRjcp/9ONPUAK6KWkzKTYPX7ui0D6l7//A5Z48fu/8/3Sl17kU75wvHj993+j
S9dbIFMAP6CpxxYS4U9KtW7PZoNfofq+ns97j1/4SwDM4vd/hK63BBisKIMtsBCwaXo4tu/QBRlgFmWZuewsMumoOcMy3lSBRRwVXYTeJV8av3u4vourAmOj
jyTdj3WidHgiK5XuI+HLU4FugghMVHZB7GIB+e2MPawgs89B4gSapOhG3viHgek84HS9UCXaOmsvrjzO3AlO979ZlMs1YL44cp6TuTFEQz+ARTH54AH8+mqS
x1/bMBNRsBYaGVQEUNGFHm2y6G1lYYDRfyTEpq0jUBw02LlzHslmP9PfByy8TRDewT8MrD/joZ/rqe4TXt3cfKHBDkj1L2WIQLycyuAnFPDiP/7n/8HMbkOa
0Fp6/uOD+Jp97fJufmGXMp4n2Qd7ebJwRz3ZHo8c5ajJoL8ATtVx3K7TngA3Gw8XTqsFxrEzbE/Gbht+uENn0FbDibPo9txOu+9OuotxezRZtJ2D+FtmJHt5
XKYk8rlOD/kc/E35HKrBQI/+EtecGdnfV9AKaKigMKJwAGmpS2jOaJEYuilUSF6gEgZW0DSQg6HHGztlrM3iYHJ6ZrMKpEs3hlQEoEqFX1Qe7+LN+YrB5vpW
jU5TOUYz7Vz7fJnPpM91h4uemndBdZOjLl5q2O8PJr1ur+uM5nLiTEaD7qgr24fpc4UxWse/+qTQIYPEJPduhDkttKtSSxO6m0tQyC68+Eso/ejCcCQZMmvl
MlZxfX0QrubRI6S0RBzwxYiZGydQLhCamEL4ssmv4BsiU8qL1LVywuCqouUIyiq6/DKOBDAw4kPnncdmmB/x8oowg7Fm5ICbHatdD4VOJSN8JErAzcZVK8Mo
49//sV6rsCV+luF7BXwqBraz8kKAgVBYAqUG3QkQoFRFjhrKdGsY9/YtCWVRh1YacKyoM9CYUSbkiCB7MwfCrav1OZwptLAKgg2H8AA0kS1DgUebUBUaMEMi
Tk7dEhMP1eNULNpiE5bxzuLxQA05PzR970+JhOQ7LVhAsqFdMbsGLOEExVZCWxwvBgDaXSUTSJbURGjYHwAs0Moe0bwXw4yA/hyWxOMnl054HbMII4GfIm+0
BtWyorFeI+vy93+QnuQB7QZ4X8w2Rsx9tnX4WlARoccOvvtLidSPakaAklRjKMnHLHl/goVQvIJ8D6eySn0efionYCO4vf5cgb0BD93JZN5zlOu0nXYHJXR3
OOoM3eGd+Kk9Soujjm7lqCiBzKZFCYslFOSjCfjm765ayO0qPuEzX0neB8S0//f/ETlz7r+OHxfsSubEh8+mnCE/0VaKqxVV69i3NgTw5OIS+bXraS6s8723
hIkkBEGx8paYIPPiElDeV670kUGXMeOD1ZSjW7lzCcfMF8UiRU7HzHzd8gOfb/+Y0Q1U6KJ7xJdnwmdiVlSM5IrnYqjVjD8j13rm+QY2DnIqDZVHx1TkcaZy
0rz9Fh+LwNDYTxyDB5fCxNa7ivx8HxslUYmsyOKameL8kAAw2XGsiPz9SLfBxVzdw6U/M5N+kzHRlGbXmmRAscCJkxVrj5vfPGZkZNae2qZHhygBr1J1I07U
DS39c0KC4E36gqZLnRxzJuO/Y+zXSfW9+Eb8q/j2xEOqbImXQGYgV1ZySUYVEC3KIZ8WC4wuICgZMSHdrmiU3S1gFA/tEtZ3ASSZwBCtbFvyY0kRWFbB92ZN
KzMHRntR4e1mP0ALGOjeD0Lc8w2Rg/3+f242vhQXKCAl2oX/l7ELLZ8BsfB0PJkYwfyQynyatL1xq4lbaL3FH4y/YEfvlcc/e1HWO5Kx4vWolV86sRS58u6C
/MWjlsvgjcQ7+WKxCH//Nz+L4anD4HFyIZceSSTc3/+NNsnzzLsl3njAmzB3sTUN9sECG5aROPcwvap3YbNuxrM7a6ImwHjPgpREHh/upilypQRe6UpkvTWW
t+RIK1u3RjcULqPLhTh8bm2twOP3aEOFsp9Hc3PcdrvvjuRkNJa9wWI8lo6cDBeyP3LcSXc4mPSHve7IkXfS3IpjzW9wlMkxwrrcRbyJu5ceOYErlDJuUb4N
jFz1f5rqkxxaTD7OC3IqWhBWyRfzyhbf9CEjumkUNzeHeEA/GTtuc4uVFPtM2nynO+hPHFfJxcTt9scLNez33PYQNwMG0OAYsGS+GKk74UTBFabV+d4t+jx9
yirwOU0YtXRYm5TbED7c3OSo9ei+yjr5ru+hsKN+eMFNSRHJNSDC0mjtBWW+TDVnzc1zf/+/8TTZVlyi7qMZP/tcMxq4VhUaoOtdAiv18UJFqGXU97Cov5e6
Su7gSDwquhFzypO1X7kCcxCjuOSlMjenMmwRul1yMma/fYcnOBEAME0CRUt8Axw/+v0fMXp5QPfSte9tBpjg9D3mgClyN7PAqL53sASSju6u4pf7aBCY2f2N
HZr/3la0s6ZMAn9j9rWW2hm4QK+eJYeJDZpp5dXk5wZK4vd/i3E7jJwqjjr//d8xX7XlNdluNktvQc4V7JK7ORGr3/+xXIpHeDblsX0E9dExvRIbUIyM2xAN
4TWp2Xj19Hujv/Emhh59xhOJ6n7EvsR0YQ7W5V5vHdCrGloDpl7UEjRiFXsX7Bui4JEcRICN/SxXi9CLYx1Qkqp+6TGKO1qMyegtdYifeU/rKLNPZxN0KeW6
BdItQ9fcBclkQiUEbjXBLgplu4y7j8tAA7Sij5lZbsBLUNEjZP6AMssoFv02xvWL6Pd/JxddAsMUDbOcYqnKGAUd2qkYU5JS1lce28SZUSy0xkgHkGA+0LUh
eu0ZxVlnCmky+7WCd3jzZKYVENRt20Wqzy21fiXLi3KWnAcYQjGtkAGsPoCUhTGUXfFRyVPejjHjER9Qo683yYj5tl57yFRGj5jLP2IFzYy18vgJYPij4+xb
b42CUxgdjeeC9TkWCb8ibnJ/ys1Hipjm8p1R6MjOZvnA0Y5mf8aPu9r1upe7B9u93NXmE99b077oniHr7NE7mzfZpXd08fy9uEAE9RbeHO9FTrs55uIHrnUs
gdqeM/fZjaBYSi82/bQQdNgvQ08oVURQRIiU0xWwkQeXJztKQeQEH3I7CslrPazkoFoymk4etEkRgm8HIUut5NAMpaIpGiYgOdD2ZDrXcq7AT3cwxUz0ipY/
qJAxX7NZtQyBoleqXKXSVXlr1pILhzLi3m2OOBPodA9T/g6uaHSb4FjKmf1uz9lLrl5g7VkdQ8MIswyGmzgzMpCSf4/DrY/J6OfnMoxOeuO2jrC4BZp410qy
goj7iaaXpFdGkuCNRRwgdMskogdsW4n7m6HklAe3VQDEfh0yCU/V6G75ev8YxCBn2nFiidDq39nHlByo/xxOpt3blsnZt4Tf2W5K8xXaaKeRYB079PZQLsIK
FmZ+jJS7uxsMdE57WuB1S5nedtX7j//1v4tv+DGt/h//63/aL3UjJa4zm2Mfa36k31ixaaV4xlwqeaSyzHgyjBqqv0AdPw5//8diwYEOPEg0oLS5LBzQw9+D
6XoOxmxmxDqULTERybhlrT0J5woueReqoTey8Hi+FOQ8xXvbjKJYiGkrElXBenmyjQNUvNBwSLV6tLhpDOjLRy8Ga7dgiMTk3ODzskIn2qePC3kBAw7WOCzK
SLBKjI7MHkXpdvH/ByerLCg=
